nix/cabal-project.nix

```diff
@@ -1,13 +1,10 @@
 # This file is part of the IOGX template and is documented at the link below:
-# https://www.github.com/input-output-hk/iogx#33-nixhaskell-projectnix
+# https://www.github.com/input-output-hk/iogx#33-nixcabal-projectnix
 
-{ inputs, inputs', meta, config, pkgs, lib }:
+{ inputs, inputs', meta, config, pkgs, l, ... }:
```

Comment:
Why not keep it "lib"? Seems clearer.

Reply:
`lib` is used so frequently that I thought it deserved a single-letter name. 
One can always do this instead:
```
import (pkgs) lib;
```

Reply:
Yes `lib` is used frequently, but I've never seen `l`, so I thought *that* was confusing.